chemprop/train/molecule_fingerprint.py

```diff
@@ -26,28 +26,9 @@ def molecule_fingerprint(args: PredictArgs, smiles: List[List[str]] = None) -> L
     num_tasks, task_names = train_args.num_tasks, train_args.task_names
```

Comment:
num_tasks or task_names are never used when outputting fingerprints (this also caused the LGTM alert) - I think we should delete this line.

Reply:
I've made now made this change